projects/plugins/jetpack/class.jetpack.php

```diff
@@ -691,7 +691,7 @@ private function __construct() {
 		require_once JETPACK__PLUGIN_DIR . 'class.jetpack-gutenberg.php';
 		add_action( 'plugins_loaded', array( 'Jetpack_Gutenberg', 'load_independent_blocks' ) );
 		add_action( 'plugins_loaded', array( 'Jetpack_Gutenberg', 'load_block_editor_extensions' ), 9 );
-		add_action( 'enqueue_block_editor_assets', array( 'Jetpack_Gutenberg', 'enqueue_block_editor_assets' ) );
+		add_action( 'enqueue_block_assets', array( 'Jetpack_Gutenberg', 'enqueue_block_editor_assets' ) );
```

Comment:
Jetpack needs to also support the previous version of WordPress (6.2) where this hook doesn't enqueue styles in the editor, so we still need `enqueue_block_editor_assets` for some scenarios.

Reply:
Both hooks were introduced in WP 5.0.0. I guess we just need to check if it breaks in older versions of WP.

Reply:
Yes, the `enqueue_block_assets` hook has been available for a while, but I think only as of WP 6.3 enqueues styles to the iframed site editor. Or did I misunderstood what changed in WP 6.3.?

Reply:
NVM, manually tested on WP 6.2 and this seems to work!